ansible/roles/base/tasks/main.yml

```diff
@@ -81,6 +80,32 @@
       - python-apt
     state: present
 
+- when: base__managed_ntpd | default(True)
+  block:
```

Comment:
why th double `_` ?
Also we should probably put all things that need a default value to `defaults/main.yml`. And now that i think about it, it would be even nicer to also put variables which have no good defaults, but are needed by the role, into `defaults/main.yml` as well. Of course these variables must be comment out. This way we have a nice overview of which variables are used by the role.